controllers/api.controller.js

```diff
@@ -12,3 +12,12 @@ exports.getTopics = (req, res, next) => {
     })
     .catch(next);
 };
+
+exports.getArticleById = (req, res, next) => {
+  const articleId = req.params.article_id;
```

Comment:
You have proven you can destructure in your tests. Keep consistent, dot notation or destructuring. Not both.